### ISSUE TITLE
Cannot grant the required permissions to uBlock Origin Lite

### ISSUE THREAD

**Issue by alleluia20** [kiwibrowser/src.next#1186]
Given that uBlock Origin is probably getting discontinued (and not updated), I tried installing uBlock Origin Lite.

Nevertheless, I can just use the basic mode of the extension, not the optimal or the complete modes, because Kiwi is not prompting for the required permissions, nor is there any way to manually grant them.

The issue has been discussed on the extension site, on https://github.com/uBlockOrigin/uBOL-home/issues/105, and the consensus is that it is a Kiwi's bug.

This problem had been reported on #1089, but the submitter closed it without the bug being dealt with. As a result, I am opening a new ticket. 

Thank you, and regards.

**Comment by trmdi** [comment]
Try Adguard extension. I've been using it for awhile without any issue.

**Comment by DeKleineKobini** [comment]
Pretty sure this is related to #211, which hasn't been responded to since years ....

**Comment by khulpu-1** [comment]
Yes, I am facing this issue too and it should be in the priorities list because it's a matter of time until uBo will stop working at all.

**Comment by trmdi** [comment]
> Yes, I am facing this issue too and it should be in the priorities list because it's a matter of time until uBo will stop working at all. It might indeed be a Kiwi problem because I can't reproduce on Edge Canary. On the other hand, I can reproduce on Lemur (the known suspect).

But why not try Adguard extension? It works perfectly.

**Comment by khulpu-1** [comment]
> But why not try Adguard extension? It works perfectly.

For now, uBo (non-Lite) works. On the long run, I prefer using uBo instead of Adguard as it's lean on resources.
